beacon-chain/state/v1/getters_state.go

```diff
@@ -24,8 +24,9 @@ func (b *BeaconState) CloneInnerState() interface{} {
 
 	b.lock.RLock()
 	defer b.lock.RUnlock()
+	// TODO: Change this to something else, not sure what yet
```

Comment:
I need to think more about this function. There will be no more inner state once we remove protobuf, so `CloneInnerState` does not make much sense anymore. Maybe we just need a `Clone` method?